src/edubot/cogs/queue.py

```diff
@@ -263,6 +264,294 @@ async def updateIndicator(self, ctx):
         # Send new indicator
         self.indicator = await ctx.channel.send(embed=embed)
 
+    async def startReviewing(self, ctx, aid):
+        if aid not in self.assignments:
+            self.assignments.append(aid)
+            self.assignments.sort()
+            await self.updateIndicator(ctx)
+        else:
+            ctx.send(f"Assignment {aid} is already being reviewed.", delete_after=5)
+
+    async def stopReviewing(self, ctx, aid):
+        if aid in self.assignments:
+            self.assignments.remove(aid)
+            await self.updateIndicator(ctx)
+        else:
+            ctx.send(f"Assignment {aid} was not being reviewed.", delete_after=5)
+
+class MultiReviewQueue(Queue):
+    qtype = 'MultiReview'
+
+    class Student:
+        def __init__(self, uid):
+            self.id = uid
+            self.aid = []
```

Comment:
The `Student` class can be improved through use of a [dataclass](https://docs.python.org/3/library/dataclasses.html#dataclasses.dataclass) from the `dataclasses` built-in module. This automatically generates some magic methods like `__init__` and `__repr__` for you 😄 . If this change is implemented, the following imports will be required:

```python
from dataclasses import dataclass, field
from typing import List
```
Also moving it out of the scope of the `MultiReviewQueue` might be useful if other classes would like to use the `Student` class later on.

```suggestion
@dataclass
class Student:
    uid: int
    aid: List[int] = dataclasses.field(default_factory=list)

class MultiReviewQueue(Queue):
    qtype = 'MultiReview'
```